# Remove new line in empty object (remove-newline-from-empty-object)

## Rule Details

This rule aims to remove newline from empty object.

Examples of **incorrect** code for this rule:

```js
const a = {

};
```

Examples of **correct** code for this rule:

```js
const a = {};
```
